junit-platform-suite/junit-platform-suite.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+	`java-library-conventions`
+}
+
+description = "JUnit Platform Suite (Aggregator)"
+
+dependencies {
+	internal(platform(project(":dependencies")))
+
+	api(platform(project(":junit-bom")))
+	api(project(":junit-platform-suite-api"))
+	api(project(":junit-platform-suite-engine"))
```

Comment:
When comparing to `junit-jupiter` you would expect so yes.

However the `junit-platform-suite` module has a transitive dependency on `opentest4j` through `junit-platform-suite-engine` -> `junit-platform-engine`. When using `runtimeOnly` this dependency is not available to compile the module.

Using `runtimeOnly` does work for `junit-jupiter` because `opentest4j` is also available through `junit-jupiter-api`. However `junit-platform-suite-api` does not have a dependency on `opentest4j`.

Reply:
I see, but `implementation` should work, right?

Reply:
Seems to work. Though the documentation suggest that this is for dependencies which are purely internal and not meant to be exposed to consumers - which should happen. I'm guessing this only applies internally to a gradle project?